README.md

```diff
@@ -1,16 +1,19 @@
 Developer guide to Tessel 2 Docs Gitbook
 ============
 
-# Tools
-Install the following command line tools using `npm i {{toolname}} -g`
```

Comment:
The way I have it setup with the `npm` scripts means no need for to globally install command line dependencies. So scripts like `gitbook serve` as `npm start` run with root privileges. 

This should make it easier for overall setup and reduce clashing global dependency versions because it's all scoped to this project directory.